conda-forge.yml

```
@@ -1,2 +1,7 @@
+build_platform:
```

Would suggest adding this to make sure the `2.x` branch is getting other migration PRs

```suggestion
bot:
  abi_migration_branches:
  - 2.x
build_platform:
```